content/contributing/project-information/release-process.md

```diff
@@ -0,0 +1,121 @@
++++
+title = "Release Process"
+insert_anchor_links = "right"
+[extra]
+weight = 3
++++
+
+Bevy uses three-months-long development cycles, delimited by a weeks-long rolling release process.
+
+This means that bevy releases a new version roughly once every three to four months. In an effort to avoid crunch and encourage sustainable development, we do not pick specific dates or deadlines.
+
+## Development Cycles
+
+During a three-month development cycle, maintainers collect important PRs and issues into a GitHub Milestone. Issues included in the milestone are given priority, and the community generally makes a best-effort attempt to merge as many of these as possible over the duration of the cycle.
+
+In the lead-up to a release (generally about about two and a half months after the previous) we begin to aggressively purge the milestone of things that seem unlikely to be ready in time. The main things that tend to remain are:
+
+- Quick and easy features under active development.
+- Fixes for important regressions or crashes.
+- Large or impressive features that only need minor polish.
+
+When the milestone is complete, we publish a release candidate (ex. `0.14.0-rc.1`) and begin the release process.
```

Comment:
Also mention that a Working Group will be created for release prep, to help organize and coordinate the process.